### ISSUE TITLE
[Adaptive Cards> ActivityUpdate.json]: 'Dates and Add a comment' options are not accessible through the swipe gesture.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#7571]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.3.1-beta.20220620.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 15.5
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.20220612.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Scenarios' control and activate it.
6. Navigate to 'ActivityUpdate.json' and activate it.
7. Navigate to the 'Date' and 'Add a Comment' field and observe the issue.

**Actual Result:**
'Date' and 'Add a Comment' options are not accessible through the swipe gesture.

**Expected Result:**
'Date' and 'Add a Comment' options should be accessible through the swipe gesture.

**Note:**
Same issue is observed with Bluetooth keyboard as well.

**User Experience:**
It will impact the all user if the any element on the screen does not accessible through the swipe gesture or keyboard as user wont be able to perform desired action.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.01.01
 

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/176178045-6b0d74c9-b123-40fb-addb-4aa02589b426.MP4

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
This is already fixed.

Please use sample app release 141 and above.

And use v1.3/Scenario/ActivityUpdateWithLables.json

**Comment by MohitTyagi13** [comment]
Issue is fixed and closed already.

Verified in latest adaptive card build Version 1.0 (2.4.0-beta.20220718.1)
